### PR TITLE
Add Collaborating with Git lesson

### PR DESCRIPTION
**Pull request by aays**
Based somewhat on the R course Git lesson. Let me know what you think of the 'owner'/'collaborator' setup I've got here - it's inspired by that one time we did that lesson for the Johnson lab.

### CODE REVIEW

File: lessons/git/collaboration/lesson.md [hunk 1]
```diff
@@ -0,0 +1,495 @@
...
+ - **Authors**: Ahmed Hasan and James Santangelo
+ - **Research field**: Biology
+ - **Lesson topic**: Collaborating with Git and GitHub
+ - **Lesson content URL**: <https://github.com/utm-coders/studyGroup/tree/gh-pages/lessons/git/collaboration>
+
+In this tutorial, we will cover using Git and GitHub for collaboration, working
+through an example collaborative workflow. This lesson is best taught with
+attendees pairing up at the start. While not necessary, it is _very highly
+recommended_ that attendees who have not attended the previous [Intro Git
+lesson][intro-git] work through the material on prior to this lesson.
```
Comment (James-S-Santangelo) [inline lessons/git/collaboration/lesson.md]: Missing 'lesson' after [intro-git]. Also, delete 'on'

File: lessons/git/collaboration/lesson.md [hunk 2]
```diff
@@ -0,0 +1,495 @@
...
+lesson.
+
+# Why collaborate with Git?
+
+In our [previous Git lesson][intro-git], we covered the philosophy and basics
+of version control with command line Git. We wrapped up the lesson by uploading
+our version-controlled folder, or _Git repository_, onto GitHub, and looked at
+how to keep the GitHub version of a repo up to date with its local copy.
+
+In addition to all that, however, Git and GitHub also offer a host of
```
Comment (James-S-Santangelo) [inline lessons/git/collaboration/lesson.md]: Delete 'also'

File: lessons/git/collaboration/lesson.md [hunk 3]
```diff
@@ -0,0 +1,495 @@
...
+incredibly powerful tools for collaborative work. Git's version-controlled
+framework offers a robust and reliable environment with which to work with
+others, where all changes to files are tracked and reviewed by other
+collaborators as they come in. While there is some initial overhead in getting
+used to the extra steps involved in Git collaboration versus working on
+real-time collaborative software such as Google Docs, the Git collaborative
+workflow ensures that all materials are backed up, well documented, and with
+all changes reviewed and approved by other collaborators. It has also made the
+development of these very workshops incredibly seamless!
```
Comment (James-S-Santangelo) [inline lessons/git/collaboration/lesson.md]: Delete 'very'

File: lessons/git/collaboration/lesson.md [hunk 4]
```diff
@@ -0,0 +1,495 @@
...
+It is worth noting that many projects (including this workshop series) are
+hosted on a decentralized repo instead of a repo owned by one person. This just
+means that adding to that repo will involve the 'collaborator' workflow instead
+of the 'owner' workflow, while the 'owner' workflow will only apply if one is a
+maintainer/admin of the decentralized repo. 
+
+# Getting started with collaboration - the 'GitHub flow'
+
+Collaboration on GitHub generally centres around the 'GitHub flow'. This makes
+use of features we've seen before, such as forks and branches, but have yet to
```
Comment (James-S-Santangelo) [inline lessons/git/collaboration/lesson.md]: They will not yet have heard of 'forks and branches', right?

**Review by James-S-Santangelo — CHANGES_REQUESTED** [review]

File: lessons/git/collaboration/lesson.md (hunk 4 above)
Comment (aays) [reply lessons/git/collaboration/lesson.md]: I meant more that they may have seen the terms last lesson (a few people asked what 'on branch master' was) but we didn't do anything with them. I've restated this to make it clearer

**Pull request merged**